src/components/AccordionItem/AccordionItem.module.css

```
@@ -0,0 +1,19 @@
+/*------------------------------------*\
+    # ACCORDION ITEM
+\*------------------------------------*/
+
+/**
+ * Accordion Item houses one Accordion Button subcomponent and respect Accordion Panel subcomponent.
```

I'm kinda confused by the wording here.
```suggestion
 * Accordion Item houses one Accordion Button subcomponent and its relevant Accordion Panel subcomponent.
```

---

meant to say "its respective" 🤦‍♂️ but down for "its relevant"

---

Either's fine 👍